examples/05_Collections/01_List.md

```diff
@@ -0,0 +1,39 @@
+# List
+
+A [List](https://kotlinlang.org/docs/reference/collections.html) is an ordered collection of items, and can be mutable or immutable. Standard library allows list creation with `listOf()`, an array list, and `mutableListOf()`, that can be rendered in a *read-only* view with the [List<T>](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/-list/index.html) interface. 
```

Comment:
Better to use the word _read-only_ instead of _immutable_ because the concept of immutability is about var/val in Kotlin. By the way, mind the correct spelling: _read-only_ (with a hyphen). Please fix all occurrences in new files.
I suggest the following text:
"A [_list_](https://kotlinlang.org/docs/reference/collections.html) is an ordered collection of items. In Kotlin, lists can be either mutable (`MutableList`) or read-only (`List`). For list creation, use the standard library functions `listOf()` for read-only lists and `mutableListOf()` for mutable lists. To prevent unwanted modifications, obtain read-only views of mutable lists by casting them to `List` ."